src/util/nondet.h

```diff
@@ -13,6 +13,25 @@ Author: Diffblue Ltd.
 #include <util/std_expr.h>
 #include <util/symbol_table.h>
 
+/// Same as \ref generate_nondet_int(
+///   const mp_integer &min_value,
```

Comment:
Does this multi-line `\ref` actually render correctly in Doxygen?

Reply:
I tried it locally and it links to the right function, though only the `generate_nondet_init` part turns into a link. Some of the parameter types (like `source_locationt`) also appear as links to the Doxygen for those types.
If there is a better / more standard way of linking between overloaded functions, please let me know.